R/EuPathDB.R

```diff
@@ -48,4 +54,16 @@ EuPathDB <- function() {
 
 availableEuPathDB <- sort(read.csv('inst/extdata/granges_metadata.csv',
                                    stringsAsFactors = FALSE)$SpeciesFull)
- 
+
+
+#' Pipe operator
+#'
+#' Shamelessly scabbed from Hadley: https://github.com/sckott/analogsea/issues/32
+#'
+#' @name %>%
+#' @rdname pipe
+#' @keywords internal
+#' @export
+#' @importFrom magrittr %>%
+#' @usage lhs \%>\% rhs
+NULL
```

Comment:
Can you explain the purpose of this entry? I'm sure you have it here for a reason, but I just can't tell from looking at this exactly what the impact is going to be..

Reply:
This is a cheesy way to avoid having to import magrittr.  When I was reading the Hadley packages book, he suggested avoiding full imports; but that is harder to accomplish for infix operators.  When I did some googling, I found the linked discussion from Hadley and copy/pasted it.